### DISCUSSION TITLE
Selection Doesn't works on touch device

### DISCUSSION THREAD

**Discussion by bajrangCoder** [sourcelair/xterm.js#4780] [Q&A] [answered]
I have created Terminal mainly for touch devices, but I have found that native browser selection doesn't work on touch devices. However, it's working smoothly with a mouse. So, I want to ask: Is this a bug in xterm.js, or is it left intentionally? I know I can implement it on my own because xterm.js provides the API.

**Comment by Tyriar** [comment, marked as answer]
This is a known issue that selection isn't great on mobile, see https://github.com/xtermjs/xterm.js/issues/3727

**Reply by bajrangCoder** [reply]
Yeah, But is there any plan to fix this issue?(As I have created a terminal plugin for Android device using xtermjs and it has approx 17k user base)
However, I have tried to do it at my own but it's not good at all

https://github.com/xtermjs/xterm.js/assets/71929976/a2281509-a032-4cb5-aaba-9e8efae2f06e

**Reply by Tyriar** [reply]
As with all feature requests, this is in the backlog and may get picked up by someone. I doubt one of the core team members would contribute this tbh, but contributions from outside are welcome.
